README.rst

```diff
@@ -110,12 +147,37 @@ first schema:
 The generated administrator password is available inside the docker
 container in an environment variable ``ADMIN_PASSWORD``.
 
-The list of private IP contact points for the application can be obtained with the following snippet:
+The list of private IP contact points for the application can be
+obtained with the following snippet:
 
 .. code-block:: bash
 
     $ aws ec2 describe-instances --region $REGION --filter 'Name=tag:Name,Values=planb-cassandra' | grep PrivateIp | sed s/[^0-9.]//g | sort -u
 
+Update of a cluster
+-------------------
+
+.. important::
+   The Jolokia port 8778 should be accessible from the Odd host. Ensure the ingress rule for your clusters security group allows connections from the Odd host.
+
+To update the Docker image or AMI you should ensured that you are logged in to your account and have SSH access to you Odd host. The following commands will allow you to update the Docker image on all nodes of the cluster `mycluster`.
+
+.. code-block:: bash
+
+    $ zaws re $ACCOUNT
+    $ piu re -O $ODDHOST $ODDHOST
+    $ ./planb.py update --cluster-name mycluster --docker-image registry.opensource.zalan.do/stups/planb-cassandra-3.0:cd-69 --region eu-central-1 -O $ODDHOST
+
```

Comment:
It only states that it will update all nodes of the cluster, but not that it is doing a rolling update. This is somehow given implicit. Nonetheless I guess it's worth to mention it.